src/rendezvous/RendezvousFlow.ts

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { UnstableValue } from "../NamespacedValue";
+
+export const SETUP_ADDITIONAL_DEVICE_FLOW_V1 = "org.matrix.msc3906.v1";
+
+export const SETUP_ADDITIONAL_DEVICE_FLOW_V2 = new UnstableValue(
+    "m.setup.additional_device.v2",
```

Comment:
(aside: why v2, given we have (presumably) never used a v1 without the `org.matrix.msc3906` unstable prefix? This is the whole point of unstable prefixes.)

Reply:
IIRC, I chose `v2` in the MSC to make it explicit that the was an earlier version of the MSC in use in the wild. I appreciate that it isn't strictly necessary.

Reply:
Or are you actually suggesting making the stable name not have the v2? If so that's a good idea.